### ISSUE TITLE
MSSQL Server Vulnerability assessment points to wrong storage_account

### ISSUE THREAD

**Issue by jleonelion** [aztfmod/terraform-azurerm-caf#280]
Small bug in the below mssql_server sub-module.

The storage account for vulnerability assessments is not correctly referenced.
Currently sets to storage account defined in `var.settings.security_alert_policy.storage_account`
https://github.com/aztfmod/terraform-azurerm-caf/blob/master/modules/databases/mssql_server/vulnerability_assessment.tf#L8-L9
```
data "azurerm_storage_account" "mssql_va" {
  count = try(var.settings.security_alert_policy.vulnerability_assessment.storage_account.key, null) == null ? 0 : 1

  name                = var.storage_accounts[var.settings.security_alert_policy.storage_account.key].name
  resource_group_name = var.storage_accounts[var.settings.security_alert_policy.storage_account.key].resource_group_name
}
```
Should reference account defined in `var.settings.security_alert_policy.vulnerability_assessment.storage_account`
```
data "azurerm_storage_account" "mssql_va" {
  count = try(var.settings.security_alert_policy.vulnerability_assessment.storage_account.key, null) == null ? 0 : 1

  name                = var.storage_accounts[var.settings.security_alert_policy.vulnerability_assessment.storage_account.key].name
  resource_group_name = var.storage_accounts[var.settings.security_alert_policy.vulnerability_assessment.storage_account.key].resource_group_name
}
```

**Comment by jleonelion** [comment]
assign to me and I can fix

**Comment by jleonelion** [comment]
closing issue.  Fix was submitted in associated PR and subsequently merged into 5.3.0 release
